### PR TITLE
Feature/etch 357 spacing

### PR DESCRIPTION
**Pull request by gavmck**
Spacing utility class.

### CODE REVIEW

File: components/composition/Spacing/Spacing.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* top */
+
+.diamond-spacing-top-xs {
+  margin-top: var(-diamond-spacing-xs);
```
Comment (DanWebb) [inline components/composition/Spacing/Spacing.css:4]: Is the single dash on these vars right?

File: components/composition/Spacing/Spacing.stories.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import '../../canvas/Card/Card';
```
Comment (DanWebb) [inline components/composition/Spacing/Spacing.stories.ts:4]: Should this be import spacing?

**Review by DanWebb — CHANGES_REQUESTED** [review]

File: components/composition/Spacing/Spacing.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+.diamond-spacing-bottom-md {
+  margin-bottom: var(-diamond-spacing);
+}
+
+.diamond-spacing-bottom-lg {
+  margin-bottom: var(-diamond-spacing-lg);
+}
+
+.diamond-spacing-bottom-xl {
```
Comment (DanWebb) [inline components/composition/Spacing/Spacing.css:41]: No left or right spacing?

File: components/composition/Spacing/Spacing.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
+/* top */
+
+.diamond-spacing-top-xs {
+  margin-top: var(-diamond-spacing-xs);
+}
+
+.diamond-spacing-top-sm {
```
Comment (DanWebb) [inline components/composition/Spacing/Spacing.css:7]: I think we should remove top spacing, only bottom should be used in templates

**Review by DanWebb — CHANGES_REQUESTED** [review]

File: components/composition/Spacing/Spacing.css (hunk 3 above)
Comment (gavmck) [reply components/composition/Spacing/Spacing.css:41]: I'm about 50/50 whether to bother. Can't remember the last time I used any, usually ends up being in a grid.

File: components/composition/Spacing/Spacing.css (hunk 4 above)
Comment (gavmck) [reply components/composition/Spacing/Spacing.css:7]: I use top quite a lot when there are conditional elements

```jsx
<diamond-card></diamond-card>
{conditional && (<diamond-card class="diamond-spacing-top-md"></diamond-card>)
```

Comment (DanWebb) [reply components/composition/Spacing/Spacing.css:7]: Or maybe not if it's a utility class and not a react thing 🤔

Comment (DanWebb) [reply components/composition/Spacing/Spacing.css:7]: Alright leave it in

File: components/composition/Spacing/Spacing.css (hunk 3 above)
Comment (DanWebb) [reply components/composition/Spacing/Spacing.css:41]: They're usually a red flag

**Review by DanWebb — APPROVED** [review]

**Pull request merged**